--- origsrc/vtg-0.7.0/afrodite/Makefile.am	2010-01-13 17:56:30.000000000 -0600
+++ src/vtg-0.7.0/afrodite/Makefile.am	2010-06-29 02:22:43.961142200 -0500
@@ -61,6 +61,7 @@ libafrodite.vala.stamp: $(libafrodite_la
 
 libafrodite_la_LDFLAGS = \
 	-version-info 0:1:0 \
+	-no-undefined \
 	$(NULL)
 
 libafrodite_la_LIBADD = \
--- origsrc/vtg-0.7.0/vtg/Makefile.am	2010-06-29 02:43:25.399148400 -0500
+++ src/vtg-0.7.0/vtg/Makefile.am	2010-06-29 02:21:50.000000000 -0500
@@ -73,7 +73,7 @@ libvtg.vala.stamp: $(libvtg_la_VALASOURC
 
 libvtg_la_LDFLAGS = \
 	$(PLUGIN_LIBTOOL_FLAGS) \
-	-module -avoid-version
+	-module -avoid-version -no-undefined
 
 libvtg_la_LIBADD = \
 	../afrodite/libafrodite.la \
